core/src/main/java/com/linecorp/armeria/server/ServerBuilder.java

```diff
@@ -364,6 +365,33 @@ public ServerBuilder port(ServerPort port) {
         return this;
     }
 
+    /**
+     * Adds a new {@link ServerPort} that listens to the loopback {@code localAddress} using the specified
+     * {@link SessionProtocol}s. Specify multiple protocols to serve more than one protocol on the same port:
+     *
+     * <pre>{@code
+     * ServerBuilder sb = Server.builder();
+     * sb.localPort(8080, SessionProtocol.HTTP, SessionProtocol.HTTPS);
+     * }</pre>
+     */
+    public ServerBuilder localPort(int port, SessionProtocol... protocols) {
+        requireNonNull(protocols, "protocols");
+        return localPort(port, ImmutableList.copyOf(protocols));
+    }
+
+    /**
+     * Adds a new {@link ServerPort} that listens to the loopback {@code localAddress} using the specified
+     * {@link SessionProtocol}s. Specify multiple protocols to serve more than one protocol on the same port:
+     *
+     * <pre>{@code
+     * ServerBuilder sb = Server.builder();
+     * sb.localPort(8080, SessionProtocol.HTTP, SessionProtocol.HTTPS);
```

Comment:
It should be a list. 😉
```suggestion
     * sb.localPort(8080, List.of(SessionProtocol.HTTP, SessionProtocol.HTTPS));
```